tests/bluetooth/tester/src/btp/btp_csip.h

```diff
@@ -0,0 +1,24 @@
+/* btp_csip.h - Bluetooth tester headers */
+
+/*
+ * Copyright (c) 2023 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/bluetooth/audio/csip.h>
+
+/* CSIP commands */
+#define BTP_CSIP_READ_SUPPORTED_COMMANDS	0x01
+struct btp_csip_read_supported_commands_rp {
+	uint8_t data[0];
+} __packed;
+
+#define BTP_CSIP_DISCOVER			0x02
+struct btp_csip_discover_cmd {
+	bt_addr_le_t address;
+} __packed;
+
+#define BTP_CSIP_START_ORDERED_ACCESS		0x03
+struct btp_csip_start_ordered_access_cmd {
+	uint8_t flags;
+} __packed;
```

Comment:
What are the `flags` used for? I don't see any uses of it in this PR.

Reply:
Just RFU. It is rare for someone to have to extend a BTP order or for others to raise objections to the change, but I was advised to add RFU flag to a command that I am not 100% sure are final. CSIP test cases have not been completed in AutoPTS yet, hence my hesitation.